pom.xml

```diff
@@ -41,6 +41,12 @@
             <organization>ATT</organization>
             <organizationUrl>http://www.att.com</organizationUrl>
         </developer>
+        <developer>
```

Comment:
Thanks for your contributions, but I cannot add you here as you don't work for AT&T.